plugins/tmux-autoattach/tmux-autoattach.plugin.sh

```diff
@@ -0,0 +1,29 @@
+# @chopnico 2021
+#
+# tmux-autoattach.plugin.sh
+#
+# A tmux plugin that will automatically attach itself to a bash session.
+
+[ -z "$OSH_PLUGIN_TMUX_AUTOATTACH_BEHAVIOR" ] && export OSH_PLUGIN_TMUX_AUTOATTACH_BEHAVIOR="detach"
```

Comment:
I feel we do not have to make it an environment variable as the variable is only used in this Bash session.

Reply:
Good point.

Reply:
Would you like this types of vars upper or lower case?

Reply:
No specific preferences, but I think it needs to be consistent with other similar variables in `oh-my-bash`, so the uppercase is fine.